clang/test/Lexer/raw-string-dlim-invalid.cpp

```diff
@@ -0,0 +1,5 @@
+// RUN: %clang_cc1 -E -fsyntax-only -verify %s
+
+// expected-error@+2{{invalid character ')' character in raw string delimiter; use PREFIX( )PREFIX to delimit raw string}}
```

Comment:
This has the old spelling of that diagnostic.

Also, might be a little nicer to combine the two tests in one file, since they're testing _almost_ the same thing.

Reply:
I did not understand what you mean by old spelling...

Reply:
Before this change, the message used to say `character ')' character`. After this change, it only says "character" once, which I think is why the test is broken: https://buildkite.com/llvm-project/clang-ci/builds/12188#018daa42-213c-4e50-8de0-7b27ec43dde0

Reply:
got it! Updated.